hail/python/hail/table.py

```diff
@@ -3212,7 +3212,40 @@ def _unlocalize_entries(self, entries_field_name, cols_field_name, col_key) -> '
 
     @staticmethod
     @typecheck(tables=sequenceof(table_type), data_field_name=str, global_field_name=str)
-    def _multi_way_zip_join(tables, data_field_name, global_field_name) -> 'Table':
+    def multi_way_zip_join(tables, data_field_name, global_field_name) -> 'Table':
```

Comment:
I don't believe the behaviour of this will change in a backwards incompatible way. The only stabilization question that I think is unresolved is do we want these parameters to have default values?

Reply:
Adding a default value is backwards compatible, so we can punt on that for now. But I also don't see any problem with just picking something reasonable like "row_fields" and "global_fields". I'm fine with either option, and ready to approve once you decide.

Reply:
leave them off for now then.